index.html

```diff
@@ -6145,7 +6145,8 @@ <h3><dfn>Element Send Keys</dfn></h3>
       If <var>multiple</var> is <code>true</code>
       <var>files</var> are be appended to <var>element</var>’s <a>selected files</a>.
 
-     <li><p><a>Fire</a> an <a><code>input</code></a> event.
+     <li><p><a>Fire</a> an <a><code>input</code></a> and
+      <a><code>change</code></a> event.
```

Comment:
This might be OK, but I think it would be better to call out the ordering explicitly:

  1. Fire an input event on element.
  2. Fire a change event on element.

Or:

  1. Fire these events in order on element:
  1.1. input
  1.2. change

We should also change this to be specific about where to fire the element (note the use of _element_ above).

Reply:
I've made those changes.